Changes

```diff
@@ -36,6 +36,10 @@ Next version (4.05.0):
    and Damien Doligez, discussion with Alain Frisch and Xavier Leroy,
    feature request from the Coq team)
 
+- PR#7172, GPR#970: add extra (ocamlc -config) options
+  int_size, word_size, ext_exe
+  (Gabriel Scherer, request by Daniel Buenzli)
```

Comment:
Gabriel, you have lost Mr. Bünzli's umlaut. I guess he's too modest to complain. Once I sign the CLA, I'll fix this sacrilege.